src/Routes/PrivateRoutes.jsx

```diff
@@ -0,0 +1,12 @@
+import { useAuth } from "../contexts/authContext";
+import { Navigate } from "react-router-dom";
+
+const PrivateRoutes = ({children}) => {
+    const { state } = useAuth();
+
+    console.log("isAuth:", state.isAuth);
+
+    return state.isAuth ? children : <Navigate to="/login" />;
```

Comment:
here add replace in navigate component 
otherwise what will happen is suppose you open /videos it will redirect to the login page but URL will remain /videos 
by adding replace URL will change to /login
<Navigate to="/login" replace />